source/cloud-native-platform/path-to-live/public-dns.html.md.erb

```diff
@@ -22,17 +22,17 @@ for a new service.
 - Under environments directory there are yaml files depicting environments that represents zones corresponding to each environment.
 There can be multiple zones in one environment,
 - Create afdverify CNAME records in the appropriate zone file for environments.
-Here is an example [Pull Request](https://github.com/hmcts/azure-public-dns/pull/43/files),
+Here is an example [Pull Request](https://github.com/hmcts/azure-public-dns/pull/1387/files),
 - Create a PR with records as per the below convention
 (make sure you replace the variables with the correct values).
 
 ```
 - name: "afdverify.${name}"
   ttl: 300
-  record: "afdverify.hmcts-${env}.azurefd.net"
+  record: "afdverify.{copy endpoint hostname from frontdoor}"
```

Comment:
how do they know which one to get, this needs to explain it more, before it was easy

Reply:
yeah it was easy before, I can insert a table with hardcoded endpoint so they can see them here?

Reply:
yeah can you add a table please and have it use the table switcher thing for CFT and SDS:

https://hmcts.github.io/cloud-native-platform/troubleshooting/#connecting-to-aks-clusters

Reply:
done